unique_test.go

```diff
@@ -11,6 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !race
 // +build !race
```

Comment:
Should we use the oldest supported version, 1.23, so we can remove the obsolete build tag?

Reply:
I would avoid forcing version bump if we do not have a good reason.